src/deploy.sh

```diff
@@ -509,10 +516,70 @@ function kernel_deploy()
   fi
 }
 
+# Deploy a package
+#
+# @flag How to display commands. See `cmd_manager` inside `src/kwlib.sh`
+# @pkg_type The type of package
+# @target VM, local or remote?
+# @remote_raw Address of remote target
+#
+function pkg_deploy()
+{
+  local flag="$1"
+  local pkg_type="$2"
+  local target="$3"
+  local remote_raw="$4"
+  local package_path
+  local target_distro
+  local host
+  local port
+
+  package_path="$(tail -1 < "$KW_LIB_DIR/PKG_INFO")"
+  package_type="$(head -1 | "$KW_LIB_DIR/PKG_INFO")"
+  package_name="$(basename "$package_path")"
+
+  install_cmd="$(pkg_get_install_command "$package_type")"
+
+  host=$(get_based_on_delimiter "$remote_raw" ":" 1)
+  port=$(get_based_on_delimiter "$remote_raw" ":" 2)
+  host=$(get_based_on_delimiter "$host" "@" 2)
+
+  case "$target" in
+    2) # LOCAL
+      install_cmd+=" $package_path"
+      cmd_manager "$flag" "$install_cmd"
+      ;;
+    3) # REMOTE
+      install_cmd+=" $REMOTE_KW_DEPLOY/$package_name"
+      cp_host2remote "$package_path" "$REMOTE_KW_DEPLOY" "$host" "$port" "" "$flag"
+      cmd_remotely "$install_cmd" "$flag" "$host" "$port" ""
+      ;;
+  esac
+
+}
+
+# This function returns the command necessary to the installation of a
+# package, depending on the package type
+#
+# @package_type The type of the package
+#
+function pkg_get_install_command()
+{
+  local package_type="$1"
+
+  case "$package_type" in
+    debian)
+      echo 'dpkg -i'
+      return
+      ;;
+  esac
+}
+
 function deploy_help()
 {
   echo -e "kw deploy|d installs kernel and modules:\n" \
-    "\tdeploy,d [--remote [REMOTE:PORT]|--local|--vm] [--reboot|-r] [--modules|-m]\n" \
+    "\tdeploy,d [--remote [REMOTE:PORT]|--local|--vm] [--reboot|-r]\n" \
+    "\t         [--modules|-m] [--pkg|-p]\n" \
```

Comment:
you forgot to add `deploy`

Reply:
Actually, why did you add this together with modules? Kw provides the option to only deploy modules, the way that you put here sounds like your pkg feature will also work for modules-update.